src/components/elements/MapPolygonPanel/MapPolygonCheckPanelItem copy.tsx

```diff
@@ -0,0 +1,209 @@
+          <Text variant="text-14-semibold" className="flex items-center">
+            <Icon name={IconNames.REQUEST} className="h-5 w-5 lg:h-6 lg:w-6" />
+            &nbsp; Request Support
+          </Text>
+        </Button>
+      )
+    },
+    {
+      id: "6",
+      render: () => (
+        <Button variant="text" onClick={openFormModalHandlerConfirm}>
+          <Text variant="text-14-semibold" className="flex items-center ">
+            <Icon name={IconNames.TRASH_PA} className="h-5 w-5 lg:h-6 lg:w-6" />
+            &nbsp; Delete Polygon
+          </Text>
+        </Button>
+      )
+    }
+  ];
+
+  const dynamicClasses = (status: string) => {
+    switch (status) {
+      case "Submitted":
+        return "bg-blue";
+      case "Approved":
+        return "bg-green";
+      case "Needs More Info":
+        return "bg-tertiary-600";
+      case "Draft":
+        return "bg-pinkCustom";
+      default:
+        return "bg-blue ";
+    }
+  };
+
+  return (
+    <div>
+      <div {...props} className={className}>
+        <div className="flex items-center gap-2">
+          <div className={classNames("h-4 w-4 rounded-full", dynamicClasses(status))} />{" "}
+          <div className="flex flex-1 flex-col">
+            <Text variant="text-14-light" className="text-white">
+              {title}
+            </Text>
+          </div>
+          <div className="flex h-full items-start self-start">
+            <When condition={!!polygon}>
+              <button
+                onClick={() => {
+                  setOpenCollapse(!openCollapse);
+                }}
+              >
+                <Icon
+                  name={IconNames.CHEVRON_DOWN}
+                  className={classNames(
+                    "h-3 w-3 rounded-lg text-white hover:fill-primary hover:text-primary lg:h-4 lg:w-4",
+                    { "rotate-180 transform": !!openCollapse }
+                  )}
+                />
+              </button>
+            </When>
+
+            <Menu container={refContainer?.current} placement={MENU_PLACEMENT_RIGHT_BOTTOM} menu={itemsPrimaryMenu}>
+              <Icon
+                name={IconNames.IC_MORE_OUTLINED}
+                className="h-4 w-4 rounded-lg text-white hover:fill-primary hover:text-primary lg:h-5 lg:w-5"
+              />
+            </Menu>
+          </div>
+        </div>
+        <When condition={!!polygon && !!openCollapse}>
+          <div className="my-3 grid gap-3 px-4">
+            {polygon?.map((item, index) => (
+              <div key={index} className="flex items-center gap-2">
+                <Icon name={IconNames.ERROR_WHITE_BORDER_RED} className="h-4 w-4 rounded-lg text-white lg:h-5 lg:w-5" />
+                <Text variant="text-14-light" className="text-white">
+                  {item}
+                </Text>
+              </div>
+            ))}
+          </div>
+        </When>
+      </div>
+    </div>
+  );
+};
+
+export default MapPolygonCheckPanelItem;
```

Comment:
Please remove ` copy` from the filename for this component.

Reply:
Actually, is this just an unused file? It looks pretty close to the same as the file below; please review and incorporate the feedback from here into the component below.

Reply:
@diego-morales-flores-1996 @dottyy

Reply:
it's a copy. He was removed